src/docs/wtperf.dox

```diff
@@ -231,7 +233,7 @@ number of tables to run operations over. Keys are divided evenly over the tables
 @par table_count_idle (unsigned int, default=0)
 number of tables to create, that won't be populated. Default 0.
 @par threads (string, default="")
-workload configuration: each 'count' entry is the total number of threads, and the 'insert',  'modify', 'read' and 'update' entries are the ratios of insert, modify, read and update  operations done by each worker thread; If a throttle value is provided each thread will do a  maximum of that number of operations per second; multiple workload configurations may be  specified per threads configuration; for example, a more complex threads configuration might be  'threads=((count=2,reads=1)(count=8,reads=1,inserts=2,updates=1))' which would create 2 threads  doing nothing but reads and 8 threads each doing 50% inserts and 25% reads and updates.   Allowed configuration values are 'count', 'throttle', 'inserts', 'reads', 'read_range',  'modify', 'modify_delta', 'modify_distribute', 'modify_force_update', 'updates',  'update_delta', 'truncate', 'truncate_pct' and 'truncate_count'. There are also behavior  modifiers, supported modifiers are 'ops_per_txn'
+workload configuration: each 'count' entry is the total number of threads, and the 'insert',  'modify', 'read' and 'update' entries are the ratios of insert, modify, read and update  operations done by each worker thread; If a throttle value is provided each thread will do a  maximum of that number of operations per second; multiple workload configurations may be  specified per threads configuration; for example, a more complex threads configuration might be  'threads=((count=2,reads=1),(count=8,reads=1,inserts=2,updates=1))' which would create 2  threads doing only reads and 8 threads each doing 50% inserts and 25% reads and updates.  Allowed configuration values are 'count', 'throttle', 'inserts', 'reads', 'read_range',  'modify', 'modify_delta', 'modify_distribute', 'modify_force_update', 'updates',  'update_delta', 'truncate', 'truncate_pct' and 'truncate_count'. There are also behavior  modifiers, supported modifiers are 'ops_per_txn'
```

Comment:
nit: there is an extra space in "update  operations"

Reply:
Done